src/draw/lv_image_decoder.h

```diff
@@ -49,6 +49,22 @@ struct _lv_image_decoder_dsc_t;
 struct _lv_image_decoder_t;
 struct _lv_cache_entry_t;
 
+/**
+ * Image decoder args.
+ * It determines how to decoder an image, e.g. whether to premultiply the alpha or not.
+ * It should be passed to lv_img_decoder_open() function. If NULL is provided, default
+ * args are used.
+ *
+ * Default args:
+ * all field are zero or false.
+ */
+typedef struct _lv_image_decoder_args_t {
+    int32_t frame_id;       /*A not use args*/
+    bool stride_align;      /*Whether stride should be aligned*/
+    bool premultiply;       /*Whether image should be premultiplied or not after decoding*/
+    bool no_cache;          /*Whether this image should be kept out of cache*/
```

Comment:
1. It can be added later too, but for vector image we can specify the target size here as well. 
2. What about a target color format option?

Reply:
For vector image, I'm afraid more `args` are needed. Let's add them when get there.

If I understand correctly, you mean a `prefered color format`? I'm not sure of how to use it for now. It could be a future feature too.

Reply:
`prefered color format` really a better name as it's just a request that might or might not be fulfilled. 

By this option I mean for an RGB565 display the user might want RGB565 decoded image too.

But it's not critical for sure.